src/components/transactionsV2/transactionsV2.css

```diff
@@ -101,3 +105,23 @@
     color: #00b262;
   }
 }
+
+.container {
+  display: flex;
+  background: #f8f9fb;
```

Comment:
Needs to add a `border-bottom: 1px solid var(--color-light-gray);`, and remove the one on the: https://github.com/LiskHQ/lisk-hub/blob/de8faa9985745b5f50dcedc292987a909e786ff1/src/components/transactionsV2/transactionsV2.css#L19